core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -297,7 +297,7 @@ querySpecification
     ;
 
 groupBy
-    : setQuantifier? groupingElement (',' groupingElement)*
+    : setQuantifier? AUTO? (groupingElement (',' groupingElement)*)?
```

Comment:
`AUTO` should be a `groupingElement`, just like `CUBE`, `ROLLUP`, etc.

Reply:
Changed to `groupingElement`.